### PR TITLE
Enhancement to teams index

### LINKED ISSUE #990: Improve Teams Index for Admins
# Feature Request

## What problem does this solve

The current layout of the teams index page does not provide much data and is hard to follow without a header row.

## Describe the solution you'd like

I would like to see the following columns:

* Balance (remaining balance)
* Paid (sum of credit transactions that are not gifted)
* Gifted (sum of credit transactions that are gifted)
* Active Campaigns (number, link to org campaigns overview)
* Total Campaigns (number, link to org campaigns overview)
* Team Members (number, link to org members page)
* End Date (end date of most recent campaign)
* Last Login (date of most recent login of members)

There should be a summary row (footer or below header).

I would also like the data to be segmented into three categories:

* Active (only show teams that have active campaigns)
* Inactive (only show teams that do not have active campaigns)
* All

## Additional context

image

## Extra Credit

It would be ideal to use [Kendo Grid](https://www.telerik.com/kendo-ui/grid) for this view. They provide column-based filtering which would solve a lot of issues not only for this table but for others as well.

### PR DESCRIPTION
**Pull request by andrewmcodes**
# Enhancement

## Description

Improve team index for admins

Fixes #990

### General

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have updated DB schema annotations or I have not made any DB schema changes

### CodeFund Core Team

- [x] I have added the appropriate milestone if applicable
- [x] I have requested a review from another developer(s)

## Screenshots

CodeFund | Ethical Advertising 2020-03-04 17-08-51

### CODE REVIEW

File: app/views/organizations/_tr.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= table_row do %>
```
Comment (andrewmcodes) [inline app/views/organizations/_tr.html.erb]: Could you help me with the cache key here @hopsoft

File: app/models/concerns/organizations/statusable.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+module Organizations
+  module Statusable
+    extend ActiveSupport::Concern
+
+    STATUSES = ENUMS::ORGANIZATION_STATUSES
+
+    included do
+      # scopes ....................................................................
+      scope :active, -> { where(id: Campaign.active.select(:organization_id)) }
+      scope :inactive, -> { where.not id: active }
```
Comment (hopsoft) [inline app/models/concerns/organizations/statusable.rb]: These are beautiful scopes.

Comment (andrewmcodes) [reply app/models/concerns/organizations/statusable.rb]: ❤️
